core-lib/src/main/java/org/idpass/smartscanner/lib/nfc/details/NFCDocumentTag.kt

```diff
@@ -137,6 +137,36 @@ class NFCDocumentTag(val readDG2: Boolean = true) {
                     additionalPersonDetails.title = dg11.title
 
                     passport.additionalPersonDetails = additionalPersonDetails
+
+                    // Hash Checking
+                    val hashCheckNotSucceeded = "hash-check not SUCCEEDED"
+                    if (verifySecurity.ht != VerificationStatus.Verdict.SUCCEEDED) {
+                        Log.e(TAG, hashCheckNotSucceeded)
```

Comment:
Need to add to sentry log if `hash-check not SUCCEEDED` log message

Reply:
Okay @typelogic will add and separate